packages/preview/vienna-tech/0.1.1/typst.toml

```diff
@@ -0,0 +1,28 @@
+# for a description of available keys, see https://github.com/typst/packages/?tab=readme-ov-file#package-format
+
+[package]
+name = "vienna-tech"
+version = "0.1.1"
+entrypoint = "src/lib.typ"
+authors = ["Niko Pikall"]
+license = "Unlicense"
+description = "An unofficial template for writing thesis at the TU Wien civil- and environmental engineering faculty."
+repository = "https://github.com/npikall/vienna-tech.git"
+keywords = ["thesis", "tuw", "TU Wien", "civil engineering", "environmental engineering"]
+categories = ["thesis"]
+exclude = [
+	".github",
+	"docs",
+	"scripts",
+	"tests",
+	".typstignore",
+	"Justfile",
```

Comment:
I recommend excluding the `thumbnail-long.png` file to keep the package size down.